### ISSUE TITLE
Guide to migrate to v4 docker image incomplete

### ISSUE THREAD

**Issue by MiniGod** [verdaccio/verdaccio#2135]
**Describe the bug**

TLDR: The "Migrating Verdaccio" blog post should be updated regarding docker volumes in v4. The "docker-examples" directory should also be updated.

I'm migrating our verdaccio docker-compose setup from v2 to v4 and after upgrading the storage directory was empty.

The following places I looked for information on how to migrate:

1. The [migration guide](https://verdaccio.org/blog/2019/02/24/migrating-verdaccio) only mentions changes to environment variables.
2. The [`docker-compose.yaml` file in the docker-examples](https://github.com/verdaccio/verdaccio/blob/master/docker-examples/v4/reverse_proxy/nginx/root_path/docker-compose.yaml) also seems unchanged since when we set it up in 2018 with verdaccio v2.
3. [The "The new Docker image for Verdaccio 4" blog post](https://verdaccio.org/blog/2019/05/13/the-new-docker-image-verdaccio-4) mentions the new volumes under "security" but not very obvious that it was breaking change and no steps to migration.

Since we use a default config with no plugins, it all looked like a very simple upgrade, but after upgrade no packages were visible (only had to rename the `-db.json` file, I thought).

I later found that this was due to the docker container for v4 uses a separate volume for `/verdaccio/storage`, so with the v4 docker image, it will not use the `storage` directory from the `/verdaccio` volume.

This is a critical documentation error that could make people lose their data, since the `/verdaccio/storage` directory is not in the volume defined in the `docker-compose.yaml`.

I had to dig through the issues to find a solution. This issue, and ultimately this comment, helped me getting `/verdaccio/storage` into v4: https://github.com/verdaccio/verdaccio/issues/1324#issuecomment-501379852

I have solved it, for me, but the documentation should be updated.

Related issues: #1274 #1324

**To Reproduce**

1. Set up verdaccio v2 (or v3?) docker using [the example](https://github.com/verdaccio/docker-examples/blob/f95db5c3b4b14663cd6093b36130c52bcb8dad7c/nginx-verdaccio/docker-compose.yaml) (change `:latest` to `:2` in `docker-compose.yaml`).
2. Publish a package
3. Open verdaccio page and see the package in the list
4. Upgrade to verdaccio v4. According to the [migration guide](https://verdaccio.org/blog/2019/02/24/migrating-verdaccio), nothing in the example above is breaking between the versions.
5. Open verdaccio page and see the package list is empty

**Expected behavior**

The migration guide should explain how to migrate the `/verdaccio/storage` data to v4 docker image.

**Configuration File (cat ~/.config/verdaccio/config.yaml)**

default config.

**Environment information**

Using docker (docker-compose).

Don't know how to run `verdaccio --info` when using docker. The old image was `:latest` tag from 28th of January 2018. I'm assuming it was v2 since it used `.sinopia-db.json`.
Upgraded to image that is tagged `:4` today.

**Comment by juanpicado** [comment]
> I have solved it, for me, but the documentation should be updated.

Seems you have good domain of the problem, would you like to PR?
https://github.com/verdaccio/website/blob/master/website/blog/2019-02-24-migrating-verdaccio.md

**Comment by mbtools** [comment]
Not relevant anymore. Should anyone be stuck on such old Verdaccio version, please reach out on [Discord](https://discord.gg/T7gJmBM6nv).
